index.js

```diff
@@ -202,6 +202,18 @@ class HyperDHT extends DHT {
     const query = this.query({ target, command: COMMANDS.MUTABLE_GET, value: c.encode(c.uint, userSeq) }, opts)
     const latest = opts.latest !== false
 
+    if (opts.value && opts.signature) {
+      if (!Persistent.verifyMutable(opts.signature, userSeq, opts.value, publicKey)) {
+        throw new Error('Invalid signature')
+      }
```

Comment:
Remove this if. The signature should always be valid - up to the user to ensure that